packages/relay/tests/lib/eth.spec.ts

```diff
@@ -2344,6 +2344,7 @@ describe('Eth', async function () {
     it('valid receipt on match', async function () {
       // mirror node request mocks
       mock.onGet(`contracts/results/${defaultTxHash}`).reply(200, defaultDetailedContractResultByHash);
+      mock.onGet(`contracts/${defaultDetailedContractResultByHash.created_contract_ids[0]}`).reply(404);
```

Comment:
All the aded flows are for 404's.
Is there a 200 case that verifies a valid `contract.evm_address`?